modules/gsp-cluster/values.tf

```diff
@@ -7,6 +7,8 @@ data "template_file" "values" {
     cluster_name                     = "${var.cluster_name}"
     cluster_domain                   = "${var.cluster_domain}"
     cluster_domain_id                = "${var.cluster_domain_id}"
+    # a list of egress IPs, where each entry is surrounded by " marks
+    egress_ip_addresses              = "\"${join("\",\"", var.egress_ips)}\""
```

Comment:
If someone can come up with a less horrific way of doing this...please shout!

Reply:
Can we use terraform's `jsonencode` like we do for the other "array of strings" values in this file (like the _arns, etc)

Reply:
`jsonencode`d